lib/spdy-transport/protocol/http2/parser.js

```diff
@@ -109,6 +111,17 @@ Parser.prototype.onFrameHead = function onFrameHead(buffer, callback) {
     id: buffer.readUInt32BE() & 0x7fffffff
   };
 
+  if (this.isServer && header.id != 0) {
```

Comment:
What about clients, should they verify that server-sent PUSH_PROMISE ids are bigger than previous?